examples/demo-stf/src/sov-cli/native.rs

```diff
@@ -0,0 +1,487 @@
+use anyhow::Context;
+use borsh::BorshSerialize;
+use clap::Parser;
+use const_rollup_config::ROLLUP_NAMESPACE_RAW;
+use sov_modules_api::transaction::Transaction;
+use sov_modules_stf_template::RawTx;
+use std::fs;
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use demo_stf::runtime::cmd_parser;
+
+use sov_modules_api::{
+    default_context::DefaultContext, default_signature::private_key::DefaultPrivateKey,
+    AddressBech32, PublicKey, Spec,
+};
+
+use demo_stf::runtime::{module_parse_helper, ModuleCommands};
+
+type C = DefaultContext;
```

Comment:
Does it mean, that cli is only available for DefaultContext? What if another context is used?